examples/inference/benchmark_llama.py

```diff
@@ -88,9 +89,10 @@ def benchmark_inference(args):
     with torch.no_grad():
         config = CONFIG_MAP[args.model]
         config.pad_token_id = config.eos_token_id
-        model = transformers.LlamaForCausalLM(config).cuda()
+        # model = transformers.LlamaForCausalLM(config).cuda()
+        model = transformers.LlamaForCausalLM.from_pretrained("/home/caidi/llama_model/").cuda()
         model = model.eval()
-        tokenizer = AutoTokenizer.from_pretrained("hf-internal-testing/llama-tokenizer")
+        tokenizer = AutoTokenizer.from_pretrained("/home/caidi/llama_model/")
```

Comment:
Do not use your actual path. Make it an argument passed via command line.

e.g. 

```python
Model.from_pretrained(args.model_path)
```